### ISSUE TITLE
Early onset or familial intestinal pseudo obstruction

### ISSUE THREAD

**Issue by andrewdevereau** [obophenotype/human-phenotype-ontology#1016]
Attached are a set of terms which have been requested when developing a disease model for this disease.  I have put these into broad groups. I think that the existing term HP:0002270 'Abnormality of the autonomic nervous system’  corresponds to the ‘visceral neuropathies’ group but there does not seem to be a similar existing term for enteric myopathies, so this may also be needed?

[New term request for HPO - Early onset or familial intestinal pseudo obstruction.docx](https://github.com/obophenotype/human-phenotype-ontology/files/422959/New.term.request.for.HPO.-.Early.onset.or.familial.intestinal.pseudo.obstruction.docx)

**Comment by pnrobinson** [comment]
Added grouping term for Abnormality of enteric nervous system morphology

**Comment by pnrobinson** [comment]
I added the term "Enteric neuronal degeneration" (HP_0025030) with definition: Deterioration of enteric neurons with impairment of enteric neuronal structure. Typical neuropathological findings include qualitative (e.g., neuronal swelling, intranuclear inclusions, axonal degeneration) and quantitative (e.g., reduction in the number of neurons) abnormalities of the enteric neurons.

I used this term name with the intention of clarifying we are talking about a phenotypic feature rather than a disease (The phrase "degenerative enteric neuropathy" is  used both as a disease diagnosis or as a pathological finding, I think). I added the latter as RELATED SYNONYM
